holoviews/plotting/widgets/widgets.js

```diff
@@ -13,14 +10,13 @@ HoloViewsWidget.prototype.init_slider = function(init_val){
 }
 
 HoloViewsWidget.prototype.populate_cache = function(idx){
-  this.cache[idx].html(this.frames[idx]);
+  this.cache[idx].innerHTML = this.frames[idx];
```

Comment:
``innerHTML`` can be quite different from ``html`` - I think something to do with escaping. So this is a bugfix?

Reply:
No, this is converting jQuery to pure JS.

Reply:
Ok. But I am not 100% sure that these two things are exactly equivalent. They may be but subtle annoyances like escaping can creep into things that look simple like this. I'm happy if it works and it probably does if the HTML isn't expected to have special characters that need escaping.